pint_xarray/checking.py

```diff
@@ -0,0 +1,213 @@
+import functools
+
+from pint import Quantity, Unit
+from xarray import DataArray, Dataset
+
+from .accessors import PintDataArrayAccessor  # noqa
+
+
+def expects(*args_units, return_units=None, **kwargs_units):
+    """
+     Decorator which ensures the inputs and outputs of the decorated function are expressed in the expected units.
```

Comment:
It looks like the indentation is slightly wrong here, causing `napoleon` to complain about unexpected section titles (I think?). Can you make sure the indentation is a multiple of 4 spaces?